packages/nuz-utils/src/checkIsProductionMode.spec.ts

```diff
@@ -0,0 +1,15 @@
+import * as func from './checkIsProductionMode'
```

Comment:
Use `checkIsProductionMode` insead of `func`.
With a file export default only don't need to use:
```js
import * as func from './checkIsProductionMode'
```
Use below:
```js
import checkIsProductionMode from './checkIsProductionMode'
```